Sources/Css/Display.swift

```diff
@@ -214,8 +214,11 @@ public func rect(top: Css.Size, right: Css.Size, bottom: Css.Size, left: Css.Siz
       [
         "rect(",
         top.value().unValue,
+        ",",
```

Comment:
forgot to intersperse commas between clip values. it's now rendering as `rect(1px1px1px1px)` causing a css error